### PR TITLE
Small changes in the README and security checks to be BSD (incl. OS X) compliant

### PR DESCRIPTION
**Pull request by AlexanderWillner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ git checkout 0.5.0
 A Vagrantfile is provided that provisions everything on a single VM. To run, first ensure that your system has 4GB of RAM free, then:
 
 ```
-sudo pip install -r requirements.txt
+sudo pip -H install -r requirements.txt
```
Comment (siddharthist) [inline README.md]: What does this flag do? It's not in my version of `pip`.

File: security-setup [hunk 2]
```diff
@@ -1,6 +1,15 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```
Comment (siddharthist) [inline security-setup]: This was changed intentionally to `python2` to comply with [PEP-0394](https://www.python.org/dev/peps/pep-0394/):
>  in preparation for an eventual change in the default version of Python, Python 2 only scripts should either be updated to be source compatible with Python 3 or else to use python2 in the shebang line.

File: README.md (hunk 1 above)
Comment (AlexanderWillner) [inline README.md]: Yeah, should have been ```sudo -H pip install -r requirements.txt```

File: README.md [hunk 3]
```diff
@@ -114,7 +114,7 @@ git checkout 0.5.0
 A Vagrantfile is provided that provisions everything on a single VM. To run, first ensure that your system has 4GB of RAM free, then:
 
 ```
-sudo pip install -r requirements.txt
+sudo -H pip install -r requirements.txt
```
Comment (siddharthist) [inline README.md]: Is this only applicable to OSX?

Comment (siddharthist) [inline README.md]: @AlexanderWillner ping

**Comment by siddharthist** [comment]
Closing due to lack of feedback, please reopen if this is still an issue.

**Pull request closed without merging**